src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java

```diff
@@ -75,6 +82,15 @@ public WorkItemsMenu clickWorkItemsMenu() {
         return new WorkItemsMenu();
     }
 
+    public SSIAHome(){
+        CommonEvents.isPresent(ssiaHomeContainer);
```

Comment:
Use "CommonEvents.isVisible(ssiaHomeContainer);" instead of "CommonEvents.isPresent(ssiaHomeContainer);"